app/workers/commit_monitor_handlers/commit/bugzilla_common/bugzilla_common.rb

```diff
@@ -0,0 +1,33 @@
+module CommitMonitorHandlers
```

Comment:
The file location of this method is weird.  I feel like it should go a lot higher...maybe into /lib/bugzilla_worker_common.rb (or `shared` or `mixin`)